examples/meals.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
Can the file `meals.go` be named something more descriptive of the feature it's demoing, such as `select_description.go`?